app/views/directives/_edit-command.html

```diff
@@ -81,14 +81,19 @@
     </span>
   </div>
   <div class="help-block">
-    Enter the command to run inside the container. The command is considered successful if its exit code is 0.
-    Drag and drop command arguments to reorder them.
+    <div ng-if="!type">
+      Enter the command to run inside the container. The command is considered successful if its exit code is 0.
+      Drag and drop command arguments to reorder them.
+    </div>
+    <div ng-if="type">
+      Enter the arguments that will be appended to the container's command. Drag and drop arguments to reorder them.
```

Comment:
This text is not necessarily always correct when the type is set. We should add a description scope value if we need to say something specific like this.